tests/ui/crashes/ice-5835.rs

```diff
@@ -0,0 +1,8 @@
+#![rustfmt::skip]
+
+pub struct Foo {
+    /// 位	
```

Comment:
I would add a comment here for people like me who's editor auto-remove trailing whitespaces.

```suggestion
    /// 位	
    ///   ^ Don't remove this tab character!
```